week-1/refactor/find.js

```diff
@@ -1,13 +1,11 @@
 // Refactor the implementation of find to use a for...of loop
 
 function find(list, target) {
-  for (let index = 0; index < list.length; index++) {
-    const element = list[index];
+  for (const [element,index] of list.entries()) {
```

Comment:
There is a ready array method doing just what you're function does.
Is the task to implement finding without using ready js functions?
The method can be found in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array

Reply:
I take help with this link thnX